shared/wallets/asset/index.js

```diff
@@ -0,0 +1,163 @@
+// @flow
+import * as React from 'react'
+import * as Types from '../../constants/types/wallets'
+import {Box2, ClickableBox, Divider, Icon, Text} from '../../common-adapters'
+import {globalMargins, platformStyles, styleSheetCreate} from '../../styles'
+
+export type Props = {
+  availableToSend: string, // non-empty only if native currency
+  balance: string,
+  code: string, // The same as `name` except for XLM
+  equivAvailableToSend: string, // non-empty only if native currency e.g. '$123.45 USD'
```

Comment:
i wonder if we want this all to be so explicit, aka, this component doesn't really care about these explicit stellar specific things. like it could just take an array of strings to draw. lets chat